.github/actions/canary/actions.yml

```diff
@@ -0,0 +1,43 @@
+name: 'Canary releases'
```

Comment:
Contender for `conda/actions`?

Reply:
I think so? I'm not sure if it's useful for other projects generally, but then again maybe that's okay. Being able to install conda-build easily would be beneficial as well.

Reply:
This could be something that would be useful for `conda-build`? Or other plugins as that becomes more of a thing?